offsite_payments

```diff
@@ -0,0 +1 @@
+Subproject commit 2bf09b604ace715ca19f2a011d42ac840726ba00
```

Comment:
This happened because `activemerchant/offsite_payments` was cloned into the `active_merchant` repo in your branch. I would suggest checking out `master` and doing a `rm -rf offsite_payments` and starting your branch over.